### DISCUSSION TITLE
How to get non-zero minimum values in simulation results

### DISCUSSION THREAD

**Discussion by anoikob** [omnetpp/omnetpp#1279] [Q&A]
Assume this result screen
![image](https://github.com/user-attachments/assets/76e23fda-5173-44e0-9e2f-b0d4e126461e)

As you can see, all the minimum values start at zero. Is there a way to have the minimum non-zero value of the experiment as the first "minimum value"? 

Edit1: I am trying to modify the vector recording intervals as shown here
![image](https://github.com/user-attachments/assets/ca76b691-1fd2-4b1e-9ce8-1e138de41d2a)

However, I get a syntax error here 
![image](https://github.com/user-attachments/assets/3a308ff5-baed-43b5-8410-78313d904481)

This line is in an .ini file, as that's where I'm assuming it's used. What's wrong?

Edit2: I managed to get rid of the error with **.vector-recording-intervals = 3.. 
However, this does not let me specify the interval as I want. I want to make the recording start at 3us, but I am still unable to find a way.

Edit3: I found the issue. The documentation is wrong. It does not let me specify the time units, but if I do something like "**.vector-recording-intervals = 0.003.." it works.

**Comment by Queliewz** [comment]
I want to ask how come your result screen shows all the values for the count and mean, while mine just appears '0' and '-' ? like this
![result 2](https://github.com/user-attachments/assets/f4be6bf1-027e-4005-88d4-ca3704d4efe0)

**Reply by anoikob** [reply]
Assuming you are using the same version as me ( 6.0.3 ), the only thing I can think of is that the vectors with no count or values are not set up correctly. Double check that you have set your vectors correctly, and that they are set to recording mode by default. Since some of the vectors did work, you could reverse engineer the problem by comparing the code of the working vectors to the code of the "dead" ones. 
I too am an amateur at this, and the reason all my vectors work is because the simulation I have created is based off of an existing showcase simulation. All I do is edit stuff and try to figure out the syntax as I go.

**Reply by anoikob** [reply]
![image](https://github.com/user-attachments/assets/5e998a3a-e129-491f-8861-353d39e32726)

This is from the tic-toc tutorial, it might help. Remember that the code shown goes into the source file of the simulation, not the ned or ini.
